examples/tutorials/curation/plot_1_automated_curation.py

```diff
@@ -0,0 +1,248 @@
+print(set(all_metric_names) == set(model.feature_names_in_))
+
+##############################################################################
+# Great! We can now use the model to predict labels. You can either pass a HuggingFace repo, or a
+# local folder containing the model file and the ``model_info.json`` file. Here, we'll pass
+# a repo. The function returns a dictionary containing a label and a confidence for each unit
+# contained in the ``sorting_analyzer``.
+
+labels = sc.auto_label_units(
+    sorting_analyzer = sorting_analyzer,
+    repo_id = "SpikeInterface/toy_tetrode_model",
+    trusted = ['numpy.dtype']
+)
+
+labels
+
+
+##############################################################################
+# The model has labelled one unit as bad. Let's look at that one, and the 'good' unit with the highest
+# confidence of being 'good'.
+
+sw.plot_unit_templates(sorting_analyzer, unit_ids=[7,9])
+
+##############################################################################
+# Nice - we see that unit 9 does look a lot more spikey than unit 7. You might think that unit
+# 7 is a real unit. If so, this model isn't good for you.
+#
+# Assess the model performance
+# ----------------------------
+#
+# To assess the performance of the model relative to human labels, we can load or generate some
+# human labels, and plot a confusion matrix of predicted vs human labels for all clusters. Here
+# we'll be a conservative human, who has labelled several units with small amplitudes as 'bad'.
+
+human_labels = ['bad', 'good', 'good', 'bad', 'good', 'bad', 'good', 'bad', 'good', 'good']
+
+# Note: if you labelled using phy, you can load the labels using:
+# human_labels = sorting_analyzer.sorting.get_property('quality')
+
+from sklearn.metrics import confusion_matrix, balanced_accuracy_score
+
+label_conversion = model_info['label_conversion']
+predictions = labels['prediction']
+
+conf_matrix = confusion_matrix(human_labels, predictions)
+
+# Calculate balanced accuracy for the confusion matrix
+balanced_accuracy = balanced_accuracy_score(human_labels, predictions)
+
+plt.imshow(conf_matrix)
+for (index, value) in np.ndenumerate(conf_matrix):
+    plt.annotate( str(value), xy=index, color="white", fontsize="15")
+plt.xlabel('Predicted Label')
+plt.ylabel('Human Label')
+plt.xticks(ticks = [0, 1], labels = list(label_conversion.values()))
+plt.yticks(ticks = [0, 1], labels = list(label_conversion.values()))
+plt.title('Predicted vs Human Label')
+plt.suptitle(f"Balanced Accuracy: {balanced_accuracy}")
+plt.show()
+
+
+##############################################################################
+# Here, there are several false positives (if we consider the human labels to be "the truth").
+#
+# Next, we can also see how the model's confidence relates to the probability that the model
+# label matches the human label
+#
+# This could be used to set a threshold above which you accept the model's classification,
```

Comment:
This is really cool, even though how it is written is quite clear, for me I had a hard time visualizing it initially.  Maybe a quick example, and motivation of the below plot? It took me a while to figure out what the plot was showing and why, a short sentence at the start of this section could make this clearer.

e.g. For example, we might accept any unit as 'good' that the model predicts as 'good' with accuracy over 80%. Otherwise, if an accuracy is lower than this, we will manually check the unit to make a decision.

Below, we will create a plot that shows ....

Reply:
Tried to clarify - it is a bit of a tough graph...